bin/bscript.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+const bscript = require('../index.js')
+const program = require('commander')
+const { version } = require('../package.json')
+
+program
+  .version(version, '-v, --version')
+
+// Assemble
+program
+  .command('assemble <raw-script>')
+  .option('-l, --literal-style [style]', 'Literal Style normal|brackets|prefixed|verbose', isString, 'normal')
+  .option('-e, --encoding [encoding]', 'Encoding ascii|base64|binary|hex|utf8', isString, 'hex')
+  .action(function (rawScript) {
+    let options = {
+      literalStyle: this.literalStyle,
+      encoding: this.encoding
+    }
+    console.log(bscript.rawToAsm(rawScript, options))
+  })
+
+// Disassemble
+program
+  .command('disassemble <asm>')
+  .option('-e, --encoding [encoding]', 'Encoding ascii|base64|binary|hex|utf8', isString, 'hex')
+  .action(function (asm) {
+    let options = {
+      encoding: this.encoding
+    }
+    console.log(bscript.asmToRaw(asm, options))
+  })
+
+// opcodes
+program
+  .command('getopcode <word>')
+  .action(function (word) {
+    console.log(bscript.opcodes.opcodeForWord(word))
+  })
+
+program
+  .command('getword <opcode>')
+  .action(function (opcode) {
+    if (isNaN(parseInt(opcode))) throw Error('<opcode> must be a single 8 bit integer')
+    console.log(bscript.opcodes.wordForOpcode(parseInt(opcode)))
+  })
+
+program
+  .command('isvalid <opcode-or-word>')
+  .action(function (value) {
+    if (!isNaN(parseInt(value))) {
+      console.log(bscript.opcodes.opcodeIsValid(parseInt(value)))
+    } else {
+      console.log(bscript.opcodes.wordIsValid(value))
+    }
+  })
+
+program
+  .command('isdisabled <opcode-or-word>')
+  .action(function (value) {
+    if (!isNaN(parseInt(value))) {
+      console.log(bscript.opcodes.opcodeIsDisabled(parseInt(value)))
+    } else {
+      console.log(bscript.opcodes.wordIsDisabled(value))
+    }
+  })
+
+program
+  .command('describe <opcode-or-word>')
+  .action(function (value) {
+    if (!isNaN(parseInt(value))) {
+      console.log(bscript.opcodes.wordForOpcode(parseInt(value)))
```

Comment:
For a PUSH_DATA opcode, wordForOpcode returns `undefined`. This should probably not be printed.